### PR TITLE
KIN-320: Improve logo and particles effect login page

### PR DESCRIPTION
**Pull request by CarlosJCD**
[KIN-320](https://loopcrack.atlassian.net/browse/KIN-320) Medium [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=loopcrack-org&utm_campaign=badge_info)

### Problem

The login views does not have the same display style as well as the particles on the background needs to be improved

### Solution

Standarization of login views and improvement of the particles effects

### Requirements checklist

Include a list of requirements necessary to consider this pull request complete

- [x] The particles effect are change into a more complex one and interacts with the user clicks
- [x] The login views must be the same, with the kinub icon on top and outside of the form card and with its own white background

### Testing

- [x] Go to the login views and see the changes ( `/login,` `/password_reset`, `/password_reset/:token` )

## Demo

[demo video](https://drive.google.com/file/d/1N0YGZOc5btoNYczxfe2Bj8GpMq5bg94u/view?usp=sharing)

[KIN-320]: https://loopcrack.atlassian.net/browse/KIN-320?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by CarlosMay7 — COMMENTED** [review]
Maybe you can reconsider the final design, we think it looks a little weird. Maybe you can add a box shadow or a sort of filter and then remove the white background if you still want to keep the logo outside of the login box

CC: @Cristianpan

File: app/Views/login/PasswordEmail.php [hunk 1]
```diff
@@ -25,24 +25,22 @@
...
                                 <div class="card-body p-4">
                                     <div class="text-center mt-2">
+                                        <div class="mb-4">
+                                            <a href="login" class="d-inline-block auth-logo ">
+                                                <img class="d-inline-block auth-logo img-thumbnail bg-transparent border-0 " src="https://www.kinub.com/images/mesa%20de%20trabajo%202.png?crc=4015297829" alt="logo kinub">
+                                            </a>
+                                        </div>
                                         <h5 class="text-primary">¿Olvidaste tu contraseña?</h5>
                                         <p class="text-muted">¡Restablécela!</p>
                                         <lord-icon src="https://cdn.lordicon.com/rhvddzym.json" trigger="loop" colors="primary:#0ab39c" class="avatar-xl">
```
Comment (Cristianpan) [inline app/Views/login/PasswordEmail.php:46]: Maybe would be good change this class `avatar-xl` for `avatar-lg`

**Review by Cristianpan — APPROVED** [review]

**Review by CarlosMay7 — APPROVED** [review]

**Review by CheloChidoone — CHANGES_REQUESTED** [review]
@CarlosJCD good job with the particles, that looks better, but for the logo did you change back what you did?, On login and password reset we see the logo still inside the box with the form, only on password reset with a token we see the logo outside but without the white background circle, let us know if we missed a step to get it to work or fix it if it is an actual mistake 
image
CC: @Oswaldo-Chan

**Comment by CarlosJCD** [comment]
> @CarlosJCD good job with the particles, that looks better, but for the logo did you change back what you did?, On login and password reset we see the logo still inside the box with the form, only on password reset with a token we see the logo outside but without the white background circle, let us know if we missed a step to get it to work or fix it if it is an actual mistake image CC: @Oswaldo-Chan

My mistake!, I totally forgot to update the response view. I will change it right away, thanks for noticing this!

**Review by Mauriciocr207 — APPROVED** [review]

**Review by CheloChidoone — APPROVED** [review]
Good Job @CarlosJCD, it looks a lot better

**Pull request merged**